src/api/kube.ts

```diff
@@ -759,6 +759,9 @@ export class KubeHelper {
       yamlCr.spec.server.cheImageTag = imageAndTag.length === 2 ? imageAndTag[1] : 'latest'
       yamlCr.spec.auth.openShiftoAuth = flags['os-oauth']
       yamlCr.spec.server.tlsSupport = flags.tls
+      if (flags.tls) {
+        yamlCr.spec.k8s.tlsSecretName = `che-tls`
```

Comment:
Where this `che-tls` value comes from?
What happens, if I put my own value in CheCluster YAML provided by parameter `--che-operator-cr-yaml`. To me it seems, that it will be overriden.

Reply:
@rhopp 

Then nothing will be overriden and additional chectl parameters will be ignored

That's the reason of line [752](https://github.com/che-incubator/chectl/pull/241/files/eae1e1a3f14078c159b5770d6ad6c3a3c3b9e413#diff-a468c62edd386b77d72847cae8716b18R752)